tests/pytests/unit/utils/test_vt.py

```diff
@@ -3,7 +3,6 @@
 import signal
 
 import pytest
-
```

Comment:
I'm reasonably sure `black` added this. Why are you removing it?

Reply:
Actually maybe it was `isort`

Reply:
It was failing the pre-commit check.